### DISCUSSION TITLE
[Question]: Detecting stationary car without events + Home Assistant?

### DISCUSSION THREAD

**Discussion by MariusMueller** [blakeblackshear/frigate#12413] [Ask A Question] [answered]
### What is your question:

Hi,

I'm using Frigate with Home Assistant which works perfectly!

I also successfully setup a driveway camera following to "how to handle parked cars", which was extremely helpful. I.e., I now receive proper events, whenever a car enters or leaves the driveway.

In Home Assistant the corresponding detection of and "car" also works ... but only as long as the event is ongoing. The "cars count" drops to zero when the event is over, and the car is still in the frame i.e. parked. I guess, this is as designt?

Nevertheless, I was wondering, if it is possible to have Home Assistant show a "cars count" of one if the car is in the frame but parked?

Thanks!

**Comment by NickM-27** [comment, marked as answer]
events are analogous to tracked objects so if the car is tracked (and thus the count is not 0) then the event is ongoing. The handling of parked cars has improved with the new UI in 0.14 so in the next version you can undo the object masks and the car count will work while also not showing the ongoing events in the UI

**Reply by MariusMueller** [reply]
Thanks for the quick response.

Looking forward to the 0.14.
